src/core/createPayload.js

```diff
@@ -10,60 +10,28 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-/* TODO: Remove.
+import createPayloadItemMerger from "./createPayloadItemMerger";
 
-  Current Request Schema:
-
-  https://git.corp.adobe.com/experience-edge/ue-gateway/
-  blob/b946662d672950898248daf346ff6adea4d41de4/resources/request.json
-
-  Top-Level nodes:
-
-  {
-    query: {},
-    "context": {
-      "identityMap": {},
-      "environment": {},
-      "webreferrer": {}
-    },
-    "events": [{ // Might contain meta per event.
-      "timestamp": 1550574782,
-      "eventId": "test",
-      "correlationID": "something",
-      "type": "::page:load",
-      "data": {}
-    }],
-    "meta": {}
-  }
-
-*/
-
-import { assign } from "../utils";
-
-const append = (content, key) => (obj = {}) => {
-  // TODO Validate.
-  console.warn(`[Payload:appendTo${key}] To Implement!`);
-  assign(content[key], obj);
-  return content;
-};
-
-export default ({
-  events = [],
-  query = {},
-  metadata = {},
-  context = {}
-} = {}) => {
-  const content = { events, query, metadata, context };
+export default () => {
+  const content = {};
```

Comment:
Maintaining the parameters here became more of a maintenance burden then they were worth (IMO) and, considering there were multiple ways of getting data onto `content` (through the constructor or through methods), we would have to wire any business logic to both sources to make sure it was properly applied. I think it makes sense to just use the exposed methods.